pg_search/src/api/operator/searchconfig.rs

```diff
@@ -33,7 +33,7 @@ use rustc_hash::FxHashSet;
 use shared::gucs::GlobalGucSettings;
 use std::ptr::NonNull;
 
-#[pg_extern(immutable, parallel_safe)]
+#[pg_extern(immutable, parallel_safe, cost = 1000000000)]
```

Comment:
Should we add a comment explaining this for posterity?

Reply:
it's here in the PR and commit message.